packages/demand-bloc-payment/src/providers/adyenErrors.ts

```diff
@@ -1,5 +1,13 @@
 import { HttpResponseOk, PaymentAuthResponse } from '@karhoo/demand-api'
 
+export class AdyenProviderError extends Error {
+  code: string
+  constructor(message: string, code: string) {
```

Comment:
Would it be more suitable for us to make `code` parameter optional?
`constructor(message: string, code?: string)`